README.md

```diff
@@ -1,18 +1,18 @@
 # Dataset Priority
 
-This section contains a relative ranking of dataset priority. 
+This section contains a relative ranking of dataset priority.
 
-While queueing is still handled manually, *this* is the single place where priority is determined. 
+While queueing is still handled manually, *this* is the single place where priority is determined.
 
-New datasets that are added should be included in this list if they are to be submitted. 
+New datasets that are added should be included in this list if they are to be submitted.
```

Comment:
Don't you need to add a listing of this in the dataset priority list? This should be at the top.